src/assets/assets.controller.ts

```diff
@@ -91,10 +91,10 @@ export class AssetsController {
       'This endpoint will provide the list of Asset holders along with their current balance',
   })
   @ApiParam({
-    name: 'ticker',
-    description: 'The ticker of the Asset whose holders are to be fetched',
+    name: 'asset',
+    description: 'The Asset (Ticker/Asset ID) whose holders are to be fetched',
     type: 'string',
-    example: 'TICKER',
+    example: '0xa3616b82e8e1080aedc952ea28b9db8b',
```

Comment:
I thought these were UUIDs now. Should this be formatted with the dashes?

Reply:
Leaving this as it is for now. Once the final decision is taken on the format to be displayed we can update all the instance